src/lib.rs

```diff
@@ -11,15 +11,19 @@
 //! [0]: https://crates.io/keywords/embedded-hal
 
 #![deny(missing_docs)]
+#![feature(doc_cfg)]
```

Comment:
is it possible to remove this? i don't think we want to stop building on stable

```
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> src/lib.rs:14:1
   |
14 | #![feature(doc_cfg)]
   | ^^^^^^^^^^^^^^^^^^^^
```

Reply:
Sure, I am afraid that `#[doc(cfg(feature = "gpio_cdev"))]` won't work then. So I guess no point in keeping them either. WDYT?

Reply:
ahh yeah, that's a bit frustrating, i think building on stable is more important for now. i'd also like to see [ci](https://github.com/rust-embedded/linux-embedded-hal/blob/master/ci/script.sh) coverage (even though it's only running cargo check).

i guess we could build both in sub-paths (under default-features gates) and choose the default export with a feature, which would put them both in the docs, not require an additional test run, and let people runtime select if required, but, seems more complicated than is probably necessary.

Reply:
Ok I have created alternative PR #30 . Let me know which one we should proceed with. Thanks